resources/src/idea/JavaActions.xml

```diff
@@ -127,7 +124,7 @@
       <add-to-group group-id="XDebugger.Frames.Tree.Popup"/>
       <add-to-group group-id="XDebugger.Actions"/>
     </action>
-    <action id="DumpThreads" class="com.intellij.debugger.actions.ThreadDumpAction" text="Get thread dump" icon="AllIcons.Actions.Dump">
+    <action id="DumpThreads" class="com.intellij.debugger.actions.ThreadDumpAction" icon="AllIcons.Actions.Dump">
```

Comment:
Not sure about this one. Why did you choose this single action (among several other ones which have the "text" attribute as well) to move to the bundle? Are you planning to reuse the text from a plugin?